# Copyright (C) 1999 Free Software Foundation, Inc.
# Ariel Bressan da Sila <ariel@conectiva.com.br>, 2000.
#
msgid ""
msgstr ""
"Project-Id-Version: gnome-db\n"
"POT-Creation-Date: 2001-09-04 02:39+0200\n"
"PO-Revision-Date: 2000-11-21 17:44-00:03\n"
"Last-Translator: Ariel Bressan da Silva <ariel@conectiva.com.br>\n"
"Language-Team: Brazilian Portuguese <ldp-br@bazar.conectiva.com.br>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8-bit\n"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: lib/gda-client/gda-connection-pool.c:257
#, c-format
msgid "could not open connection to %s"
msgstr "não pôde abrir conexão para %s"

# fe/fe_main.c:205
#: lib/gda-client/gda-connection-pool.c:269
#, c-format
msgid "Data source %s not found"
msgstr "Fonte de dados %s não encontrada"

#: lib/gda-client/gda-connection.c:428
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Base de dados '%s' não encontrada na configuração do sistema"

#: lib/gda-client/gda-connection.c:431 lib/gda-client/gda-connection.c:447
msgid "[GDA Client Library]"
msgstr ""

#: lib/gda-client/gda-connection.c:446
msgid "Could not open CORBA factory"
msgstr "Não pôde abrir fábrica CORBA"

#: lib/gda-common/gda-corba.c:73
#, c-format
msgid "CORBA System exception: %s"
msgstr "Exceção do sistema CORBA: %s"

#: lib/gda-common/gda-corba.c:108
msgid "True"
msgstr "Verdadeiro"

#: lib/gda-common/gda-corba.c:109
msgid "False"
msgstr "Falso"

#: lib/gda-common/gda-error.c:121
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s Ocorreu um erro no sistema CORBA."

#: lib/gda-common/gda-error.c:132
#, fuzzy
msgid "[CORBA User Exception]"
msgstr "Exceção de usuário CORBA: %s"

#: lib/gda-common/gda-error.c:167
msgid "Unknown CORBA exception for connection"
msgstr ""

# gda-clnt/gda-field.c:95
#: lib/gda-common/gda-field.c:640
msgid "<NULL>"
msgstr "<NULL>"

# gda-clnt/gda-field.c:101 gda-clnt/gda-field.c:103
#: lib/gda-common/gda-field.c:644
msgid "<Unknown GDA Type(NULL)>"
msgstr "<GDA desconhecido digite(NULL)>"

# gda-clnt/gda-field.c:117
#: lib/gda-common/gda-field.c:651
msgid "TRUE"
msgstr "VERDADEIRO"

# gda-clnt/gda-field.c:119
#: lib/gda-common/gda-field.c:653
msgid "FALSE"
msgstr "FALSO"

#: lib/gda-common/gda-init.c:35
msgid "Attempt to initialize an already initialized client"
msgstr "Tentativa de inicializar um cliente já inicializado"

#: lib/gda-common/gda-init.c:44 lib/gda-server/gda-server-init.c:118
#, fuzzy
msgid "Could not initialize Bonobo"
msgstr "Não pôde inicializar ct_diag()"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Salvando log...x"

#. compose message
#: lib/gda-common/gda-log.c:67
msgid "ERROR: "
msgstr "ERRO: "

#: lib/gda-common/gda-log.c:67
msgid "MESSAGE: "
msgstr "MENSAGEM: "

#: lib/gda-common/gda-util.c:113
#, fuzzy, c-format
msgid "Could not open file %s"
msgstr "Não pôde definir local para '%s'"

#: lib/gda-common/gda-util.c:138
#, fuzzy, c-format
msgid "Could not create file %s"
msgstr "não pôde alocar campo"

#: lib/gda-common/gda-xml-database.c:168
msgid "bad formed document"
msgstr ""

#: lib/gda-common/gda-xml-database.c:270
#, c-format
msgid "table %s already exists"
msgstr "tabela %s já existe"

#: lib/gda-common/gda-xml-database.c:452
#, c-format
msgid "%p is not a valid table"
msgstr "%p não é uma tabela válida"

# fe/fe_main.c:205
#: lib/gda-common/gda-xml-item.c:371
#, fuzzy, c-format
msgid "Item with id %s not found"
msgstr "Fonte de dados %s não encontrada"

#: lib/gda-server/gda-server-connection.c:392
#, fuzzy
msgid "Connection being closed"
msgstr "Conexão parece estar morta"

#: lib/gda-server/gda-server-connection.c:394
msgid "This connection is being closed, so all listeners are released"
msgstr ""

#: lib/gda-server/gda-server-connection.c:817
msgid "Could not get pointer to server implementation"
msgstr "Não pôde obter ponteiro para implementação do servidor"

#: lib/gda-server/gda-server-init.c:43 lib/gda-server/gda-server-init.c:49
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Sinal recebido %d, desligando"

#: lib/gda-server/gda-server-init.c:53
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Sinal recebido %d. Ignorando"

#: lib/gda-server/gda-server-init.c:102
msgid "Attempt to initialize an already initialized provider"
msgstr "Tentativa de inicializar um provedor já inicializado"

#: lib/gda-server/gda-server-recordset.c:53
#: lib/gda-server/gda-server-recordset.c:61
#: lib/gda-server/gda-server-recordset.c:101
#: lib/gda-server/gda-server-recordset.c:110
#, c-format
msgid "%s not implemented"
msgstr "%s não implementado"

#: lib/gda-server/gda-server.c:130
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Iniciando provedor %s sem implementação de funções"

#: lib/gda-server/gda-server.c:214
#, c-format
msgid "CORBA system exception %s"
msgstr "Exceção do sistema CORBA %s"

#: lib/gda-server/gda-server.c:218
#, c-format
msgid "CORBA user exception: %s"
msgstr "Exceção de usuário CORBA: %s"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-default-server/GNOME_Database_Provider_Default.server.in.h:1
#, fuzzy
msgid "Default provider connection factory"
msgstr "não pôde abrir conexão para %s"

#: providers/gda-default-server/GNOME_Database_Provider_Default.server.in.h:2
msgid "GDA Default Datasource Access"
msgstr ""

#: providers/gda-default-server/gda-default-connection.c:127
msgid "You must specify a directory for the database to be stored"
msgstr ""

#: providers/gda-default-server/gda-default-connection.c:235
#: providers/gda-mysql-server/gda-mysql-connection.c:282
#: providers/gda-oracle-server/gda-oracle-connection.c:334
#: providers/gda-postgres-server/gda-postgres-connection.c:487
#, fuzzy
msgid "Unknown schema type"
msgstr "ORACLE desconhecido digite %ld"

#: providers/gda-default-server/main-default.c:69
#, fuzzy
msgid "Could not register Default provider implementation"
msgstr "Não pôde registrar implementação de provedor de mensagens"

#: providers/gda-ldap-server/GNOME_Database_Provider_LDAP.server.in.h:1
msgid "GDA Datasource Access for LDAP"
msgstr ""

#: providers/gda-ldap-server/GNOME_Database_Provider_LDAP.server.in.h:2
msgid "LDAP provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:288
#: providers/gda-ldap-server/gda-ldap-connection.c:293
#: providers/gda-mdb-server/gda-mdb-connection.c:372
#: providers/gda-mysql-server/gda-mysql-connection.c:486
#: providers/gda-odbc-server/gda-odbc-connection.c:624
#: providers/gda-odbc-server/gda-odbc-connection.c:657
#: providers/gda-oracle-server/gda-oracle-connection.c:479
#: providers/gda-postgres-server/gda-postgres-connection.c:768
#: providers/gda-sybase-server/gda-sybase-error.c:127
#: providers/gda-tds-server/gda-tds-error.c:123
#, c-format
msgid "error '%s' at %s"
msgstr "erro '%s' em %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:294
#: providers/gda-ldap-server/gda-ldap-connection.c:298
#: providers/gda-ldap-server/gda-ldap-connection.c:299
#: providers/gda-mdb-server/gda-mdb-connection.c:377
#: providers/gda-mysql-server/gda-mysql-connection.c:491
#: providers/gda-mysql-server/gda-mysql-connection.c:492
#: providers/gda-odbc-server/gda-odbc-connection.c:626
#: providers/gda-odbc-server/gda-odbc-connection.c:628
#: providers/gda-odbc-server/gda-odbc-connection.c:659
#: providers/gda-odbc-server/gda-odbc-connection.c:661
#: providers/gda-oracle-server/gda-oracle-connection.c:484
#: providers/gda-oracle-server/gda-oracle-connection.c:485
#: providers/gda-postgres-server/gda-postgres-connection.c:776
#: providers/gda-postgres-server/gda-postgres-connection.c:777
#: providers/gda-primebase-server/gda-primebase-connection.c:375
#: providers/gda-primebase-server/gda-primebase-connection.c:376
#: providers/gda-sybase-server/gda-sybase-error.c:136
#: providers/gda-sybase-server/gda-sybase-error.c:137
#: providers/gda-tds-server/gda-tds-error.c:132
#: providers/gda-tds-server/gda-tds-error.c:133
msgid "Not available"
msgstr "Não disponível"

#: providers/gda-interbase-server/gda-interbase-connection.c:295
#: providers/gda-ldap-server/gda-ldap-connection.c:300
#: providers/gda-mdb-server/gda-mdb-connection.c:378
#: providers/gda-mysql-server/gda-mysql-connection.c:493
#: providers/gda-oracle-server/gda-oracle-connection.c:486
#: providers/gda-postgres-server/gda-postgres-connection.c:778
#: providers/gda-primebase-server/gda-primebase-connection.c:377
#: providers/gda-sybase-server/gda-sybase-error.c:138
#: providers/gda-tds-server/gda-tds-error.c:134
msgid "error"
msgstr "erro"

#: providers/gda-ldap-server/main-ldap.c:73
msgid "Could not register LDAP provider implementation"
msgstr "Não pôde registrar implementação de provedor LDAP"

#: providers/gda-interbase-server/GNOME_Database_Provider_InterBase.server.in.h:1
msgid "GDA Datasource Access for Interbase"
msgstr ""

#: providers/gda-interbase-server/GNOME_Database_Provider_InterBase.server.in.h:2
msgid "InterBase provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:286
#, c-format
msgid "Error code %ld"
msgstr "Erro de código %ld"

#: providers/gda-interbase-server/main-interbase.c:74
msgid "Could not register INTERBASE provider implementation"
msgstr "Não pôde registrar implementação de provedor INTERBASE"

#: providers/gda-mail-server/GNOME_Database_Provider_Mail.server.in.h:1
msgid "GDA Datasource Access for Mail"
msgstr ""

#: providers/gda-mail-server/GNOME_Database_Provider_Mail.server.in.h:2
msgid "Mail provider connection factory"
msgstr ""

#: providers/gda-mail-server/gda-mail-connection.c:107
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-sybase-server/gda-sybase-connection.c:285
#: providers/gda-tds-server/gda-tds-connection.c:278
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "SCHEMA_QTYPE %d não-manipulada"

#: providers/gda-mail-server/main-mail.c:74
msgid "Could not register Mail provider implementation"
msgstr "Não pôde registrar implementação de provedor de mensagens"

#: providers/gda-mdb-server/GNOME_Database_Provider_MDB.server.in.h:1
msgid "GDA Datasource Access for MDB (MS Access) files"
msgstr ""

#: providers/gda-mdb-server/GNOME_Database_Provider_MDB.server.in.h:2
msgid "MDB provider connection factory"
msgstr ""

# fe/fe_editor.c:185
#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:126
#: providers/gda-sybase-server/gda-sybase-command.c:141
#: providers/gda-tds-server/gda-tds-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "executando comando '%s'"

# fe/fe_editor.c:185
#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:131
#, c-format
msgid "info '%s'"
msgstr "info '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Você não pode fornecer um UNIX_SOCKET se você fornecer também uma "
"MÁQUINA ou uma PORTA. Favor remover o  UNIX_SOCKET, ou remova ambas a "
"MÁQUINA e a PORTA."

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr ""
"%s:%d: Você deve fornecer o nome de arquivo com a base da dadosMS de acesso"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Abrindo conexão com o nome de arquivo=%s, usuário=%s, senha=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:427
msgid "mysql_recset is NULL"
msgstr "mysql_recset é NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:469
#: providers/gda-mysql-server/gda-mysql-connection.c:838
#: providers/gda-sybase-server/gda-sybase-connection.c:464
#: providers/gda-tds-server/gda-tds-connection.c:448
msgid "No table name given for SCHEMA_COLS"
msgstr "Nome de tabela não dado para SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:72
#: providers/gda-mysql-server/main-mysql.c:74
msgid "Could not register MySQL provider implementation"
msgstr "Não pôde registrar implementação de provedor MySQL"

#: providers/gda-mysql-server/GNOME_Database_Provider_MySQL.server.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr ""

#: providers/gda-mysql-server/GNOME_Database_Provider_MySQL.server.in.h:2
msgid "MySQL provider connection factory"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:174
#, fuzzy
msgid ""
"You cannot provide a UNIX_SOCKET if you also provide either a HOST or a "
"PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT options"
msgstr ""
"%s:%d: Você não pode fornecer um UNIX_SOCKET se você fornecer também uma "
"MÁQUINA ou uma PORTA. Favor remover o  UNIX_SOCKET, ou remova ambas a "
"MÁQUINA e a PORTA."

#: providers/gda-mysql-server/gda-mysql-connection.c:190
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, unix_socket=%"
"s"
msgstr ""
"Abrindo conexão com usuário=%s, senha=%s, máquina=%s, porta=%s, unix_socket=%"
"s"

#: providers/gda-mysql-server/gda-mysql-connection.c:616
msgid "Unknown"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:649
msgid "No MYSQL_Connection for GdaServerConnection"
msgstr ""

#: providers/gda-odbc-server/GNOME_Database_Provider_ODBC.server.in.h:1
msgid "GDA Datasource Access for ODBC"
msgstr ""

#: providers/gda-odbc-server/GNOME_Database_Provider_ODBC.server.in.h:2
msgid "ODBC provider connection factory"
msgstr ""

#: providers/gda-odbc-server/gda-odbc-connection.c:277
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d não manipulado\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:641
#: providers/gda-odbc-server/gda-odbc-connection.c:642
#: providers/gda-odbc-server/gda-odbc-connection.c:643
#: providers/gda-odbc-server/gda-odbc-connection.c:674
#: providers/gda-odbc-server/gda-odbc-connection.c:675
#: providers/gda-odbc-server/gda-odbc-connection.c:676
msgid "no text"
msgstr "sem texto"

#: providers/gda-odbc-server/main-odbc.c:73
msgid "Could not register ODBC provider implementation"
msgstr "Não pôde registrar implementação de provedor ODBC"

#: providers/gda-oracle-server/GNOME_Database_Provider_Oracle.server.in.h:1
msgid "GDA Datasource Access for Oracle"
msgstr ""

#: providers/gda-oracle-server/GNOME_Database_Provider_Oracle.server.in.h:2
msgid "Oracle provider connection factory"
msgstr ""

# fe/fe_log.c:92
#: providers/gda-oracle-server/gda-oracle-command.c:191
msgid "Could not allocate statement handle"
msgstr "Não pôde alocar manipulador de instrução"

#: providers/gda-oracle-server/gda-oracle-command.c:197
msgid "command object being created without associated connection"
msgstr "objeto de comando sendo criado sem conexão associada"

# fe/fe_editor.c:185
#: providers/gda-oracle-server/gda-oracle-command.c:238
#, c-format
msgid "Executing command '%s'"
msgstr "Executando comando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:117
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "erro no OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:123
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "erro no OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:129
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "erro no OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:418
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "ORACLE desconhecido digite %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:118
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "ORACLE desconhecido digite em preencher_campo_valores"

#: providers/gda-oracle-server/gda-oracle-recordset.c:156
msgid "recordset object being created without associated connection"
msgstr "objeto de registro sendo criado sem conexão associada"

#: providers/gda-oracle-server/main-oracle.c:69
msgid "Could not register ORACLE provider implementation"
msgstr "Não pôde registrar implementação de provedor ORACLE"

#: providers/gda-postgres-server/GNOME_Database_Provider_Postgres.server.in.h:1
msgid "GDA Datasource Access for Postgres"
msgstr ""

#: providers/gda-postgres-server/GNOME_Database_Provider_Postgres.server.in.h:2
msgid "Postgres provider connection factory"
msgstr ""

#: providers/gda-postgres-server/gda-postgres-connection.c:889
#: providers/gda-postgres-server/gda-postgres-connection.c:1006
msgid "postgres_recset is NULL"
msgstr "postgres_recset é NULL"

#: providers/gda-postgres-server/main-postgres.c:71
msgid "Could not register PostgreSQL provider implementation"
msgstr "Não pôde registrar implementação do provedor PostgreSQL"

#: providers/gda-sybase-server/GNOME_Database_Provider_Sybase.server.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr ""

#: providers/gda-sybase-server/GNOME_Database_Provider_Sybase.server.in.h:2
msgid "Sybase provider connection factory"
msgstr ""

#.      gda_server_error_make(error, 0, cnc, "connection seems to be dead");
#: providers/gda-sybase-server/gda-sybase-command.c:136
#: providers/gda-tds-server/gda-tds-command.c:116
msgid "Connection seems to be dead"
msgstr "Conexão parece estar morta"

#: providers/gda-sybase-server/gda-sybase-command.c:153
#: providers/gda-tds-server/gda-tds-command.c:130
msgid "Filling command structure failed\n"
msgstr "Preenchimento de estrutura de comando falhou\n"

#: providers/gda-sybase-server/gda-sybase-command.c:162
#: providers/gda-tds-server/gda-tds-command.c:137
msgid "sending cmd failed"
msgstr "enviando cmd falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:169
#: providers/gda-tds-server/gda-tds-command.c:144
msgid "Allocating recordset failed"
msgstr "Alocação de registro falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:181
#: providers/gda-tds-server/gda-tds-command.c:153
msgid "Getting recordset userdata failed"
msgstr "Obtenção de dados de usuário de registro falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:241
#: providers/gda-tds-server/gda-tds-command.c:210
msgid "Command execution finished, all results processed"
msgstr "Execução de comando completa, todos os resultados processados"

# fe/fe_editor.c:138
#: providers/gda-sybase-server/gda-sybase-command.c:245
#: providers/gda-tds-server/gda-tds-command.c:214
msgid "Command failed"
msgstr "Comando falhou"

#: providers/gda-sybase-server/gda-sybase-command.c:250
#: providers/gda-tds-server/gda-tds-command.c:219
msgid "Execution of resultless command succeeded"
msgstr "Execução de comando sem resultados bem sucedida"

#: providers/gda-sybase-server/gda-sybase-command.c:268
#: providers/gda-tds-server/gda-tds-command.c:237
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() retornados %x"

#: providers/gda-sybase-server/gda-sybase-command.c:280
#: providers/gda-tds-server/gda-tds-command.c:249
msgid "No further results, command completly succeeded."
msgstr "Sem mais resultados, comando completamente bem sucedido"

#: providers/gda-sybase-server/gda-sybase-command.c:285
#: providers/gda-tds-server/gda-tds-command.c:254
msgid "An error occured during command execution."
msgstr "Ocorreu um erro durante a execução do comando."

#: providers/gda-sybase-server/gda-sybase-command.c:291
#: providers/gda-tds-server/gda-tds-command.c:258
msgid "Cmd execution returned unknown result, may be an error"
msgstr "Execução do comando retornou resultado desconhecido, podeser um erro"

#: providers/gda-sybase-server/gda-sybase-command.c:304
#: providers/gda-tds-server/gda-tds-command.c:269
msgid "No data returned, freeing recordset structure"
msgstr "Sem dados retornados, liberando estrutura de gravação"

#: providers/gda-sybase-server/gda-sybase-command.c:336
#: providers/gda-sybase-server/gda-sybase-command.c:348
#: providers/gda-tds-server/gda-tds-command.c:300
#: providers/gda-tds-server/gda-tds-command.c:310
msgid "Freeing command structure failed"
msgstr "Falhou ao liberar estrutura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:171
msgid "Not enough memory for sybase_Connection"
msgstr "Sem memória suficiente para conexão com o sybase"

#: providers/gda-sybase-server/gda-sybase-connection.c:453
#: providers/gda-tds-server/gda-tds-connection.c:437
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: tabela_nome '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:470
#: providers/gda-tds-server/gda-tds-connection.c:454
msgid "Could not allocate command structure"
msgstr "Não pôde alocar estrutura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:493
#: providers/gda-tds-server/gda-tds-connection.c:477
msgid "Could not allocate enough memory for query"
msgstr "Não pôde alocar memória suficiente para consulta"

#.          gda_server_error_make(error, NULL, cnc, __PRETTY_FUNCTION__);
#: providers/gda-sybase-server/gda-sybase-connection.c:508
#: providers/gda-tds-server/gda-tds-connection.c:492
msgid "Could not get recordset in schema_cols"
msgstr "Não pôde obter registro em schema_cols"

#: providers/gda-sybase-server/gda-sybase-connection.c:557
#: providers/gda-tds-server/gda-tds-connection.c:541
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: limite inválido, digite %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:567
#: providers/gda-tds-server/gda-tds-connection.c:551
msgid "Could not allocate cmd structure"
msgstr "Não pôde alocar estrutura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:633
#: providers/gda-tds-server/gda-tds-connection.c:617
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: limite inválido, digite %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:695
#: providers/gda-tds-server/gda-tds-connection.c:679
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Limite inválido ou não suportado, digite: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:738
#: providers/gda-tds-server/gda-tds-connection.c:722
msgid "Could not allocate GdaServerCommand"
msgstr "Não pôde alocar comando GdaServer"

#: providers/gda-sybase-server/gda-sybase-connection.c:769
#: providers/gda-sybase-server/gda-sybase-connection.c:774
#: providers/gda-tds-server/gda-tds-connection.c:752
#: providers/gda-tds-server/gda-tds-connection.c:757
#, c-format
msgid "connection status: %x"
msgstr "status da conexão: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:770
#: providers/gda-tds-server/gda-tds-connection.c:753
#, c-format
msgid "return code: %x"
msgstr "código de retorno: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:808
#: providers/gda-tds-server/gda-tds-connection.c:788
msgid "attempting reconnection"
msgstr "tentando reconectar"

#: providers/gda-sybase-server/gda-sybase-connection.c:938
#, c-format
msgid "Reconnected to '%s'"
msgstr "Reconectado para '%s'"

# fe/fe_connection.c:46
#: providers/gda-sybase-server/gda-sybase-connection.c:940
#: providers/gda-tds-server/gda-tds-connection.c:890
#, c-format
msgid "Connected to %s"
msgstr "Conectado em %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:952
#: providers/gda-tds-server/gda-tds-connection.c:904
#, c-format
msgid "Selected database '%s'"
msgstr "Base da dos selecionada '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:981
#: providers/gda-tds-server/gda-tds-connection.c:930
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Não pôde alocar estrutura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:990
#: providers/gda-tds-server/gda-tds-connection.c:939
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Não pôde alocar memória para cadeia de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:999
#: providers/gda-tds-server/gda-tds-connection.c:947
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1010
#: providers/gda-tds-server/gda-tds-connection.c:957
#, c-format
msgid "%s: sending command failed"
msgstr "%s: envio de comando falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1021
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: seleção de base de dados falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1038
#, c-format
msgid "%s: use command failed"
msgstr "%s: uso de comando falhou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1066
#: providers/gda-tds-server/gda-tds-connection.c:1020
msgid "Could not allocate locale structure"
msgstr "Não pôde alocar estrutura local"

#: providers/gda-sybase-server/gda-sybase-connection.c:1072
#: providers/gda-tds-server/gda-tds-connection.c:1026
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Não pôde definir local para '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1079
#: providers/gda-tds-server/gda-tds-connection.c:1033
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Não pôde configurar local para ser '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1085
#: providers/gda-tds-server/gda-tds-connection.c:1039
msgid "Could not drop locale structure"
msgstr "Não pôde deixar estrutura local"

#: providers/gda-sybase-server/gda-sybase-connection.c:1099
#: providers/gda-tds-server/gda-tds-connection.c:1052
#, c-format
msgid "Fatal error: %s\n"
msgstr "Erro fatal: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:73
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:85
#: providers/gda-tds-server/gda-tds-error.c:81
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Sinalizador de erro não definido, mas %s é chamado. Isto poderá ser uma falha"

#: providers/gda-sybase-server/gda-sybase-error.c:161
msgid "cs_diag() initialized"
msgstr "cs_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:165
msgid "Could not initialize cs_diag()"
msgstr "Não pôde inicializar cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:175
msgid "ct_diag() initialized"
msgstr "ct_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:179
msgid "Could not initialize ct_diag()"
msgstr "Não pôde inicializar ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:215
#, c-format
msgid "%s could not find server implementation"
msgstr "%s não pôde encontrar implementação de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:222
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s não pôde obter dado de usuário cnc"

#: providers/gda-sybase-server/gda-sybase-error.c:234
msgid "Could not get # of messages"
msgstr "Não pôde obter # de mensagens"

#: providers/gda-sybase-server/gda-sybase-error.c:241
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d ctlib mensagens a serem recuperadas"

#: providers/gda-sybase-server/gda-sybase-error.c:246
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d ctlib mensagem a ser recuperada"

#: providers/gda-sybase-server/gda-sybase-error.c:284
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "erro é nullpointer no gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:310
#: providers/gda-sybase-server/gda-sybase-error.c:382
#: providers/gda-sybase-server/gda-sybase-error.c:429
msgid "Could not get # of server messages"
msgstr "Não pôde obter # de mensagens do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:318
#, c-format
msgid "Fetching %d client messages"
msgstr "Recuperando %d mensagens de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:321
#, c-format
msgid "Fetching %d client message"
msgstr "Recuperando %d mensagem de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:330
#, c-format
msgid "Could not get client msg # %d"
msgstr "Não pôde obter mensagem de cliente # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:349
msgid "Could not get # of client messages"
msgstr "Não pôde obter # de mensagens de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:357
msgid "Could not clear client messages"
msgstr "Não pôde apagar mensagens de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:390
#, c-format
msgid "Fetching %d server messages"
msgstr "Recuperando %d mensagens de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:393
#, c-format
msgid "Fetching %d server message"
msgstr "Recuperando %d mensagem de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:402
#, c-format
msgid "Could not get server msg # %d"
msgstr "Não pôde obter msg de servidor # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:437
msgid "Could not clear server messages"
msgstr "Não pôde apagar mensagens de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:459
#: providers/gda-sybase-server/gda-sybase-error.c:499
msgid "Could not get # of cslib messages"
msgstr "Não pôde obter # de mensagens da cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:467
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Recuperando %d mensagens cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:470
#, fuzzy, c-format
msgid "Fetching %d cslib message"
msgstr "Recuperando %d mensagens cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:480
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Não pôde obter msg cslib # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:507
msgid "Could not clear cslib messages"
msgstr "Não pôde apagar mensagens da cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:641
#: providers/gda-tds-server/gda-tds-error.c:269
msgid "Server message:"
msgstr "Mensagem de servidor:"

#: providers/gda-sybase-server/gda-sybase-error.c:649
#: providers/gda-tds-server/gda-tds-error.c:277
msgid "Client message:"
msgstr "Mensagem de cliente:"

#: providers/gda-sybase-server/gda-sybase-error.c:656
#: providers/gda-tds-server/gda-tds-error.c:284
msgid "CS-Library error:"
msgstr "Erro da CS-Library:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:198
#: providers/gda-tds-server/gda-tds-recordset.c:194
#, c-format
msgid "fetch error at row %d"
msgstr "recuperar erro na fila %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:205
#: providers/gda-tds-server/gda-tds-recordset.c:201
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA na fila %d; cancelando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:213
#: providers/gda-tds-server/gda-tds-recordset.c:208
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Erro (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:386
#: providers/gda-tds-server/gda-tds-recordset.c:382
msgid "Failed fetching # of columns"
msgstr "Falhou ao recuperar # de colunas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:393
#: providers/gda-tds-server/gda-tds-recordset.c:390
#, c-format
msgid "Counted %d columns"
msgstr "Contadas %d colunas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:400
#: providers/gda-tds-server/gda-tds-recordset.c:397
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s não pôde alocar informação datafmt"

#: providers/gda-sybase-server/gda-sybase-recordset.c:410
#: providers/gda-tds-server/gda-tds-recordset.c:407
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s não pôde alocar campos de dados"

#: providers/gda-sybase-server/gda-sybase-recordset.c:428
#: providers/gda-tds-server/gda-tds-recordset.c:425
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe na coluna 3 %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:435
#: providers/gda-tds-server/gda-tds-recordset.c:432
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe na coluna # %d falhou"

#: providers/gda-sybase-server/gda-sybase-recordset.c:442
#: providers/gda-tds-server/gda-tds-recordset.c:439
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:454
#: providers/gda-tds-server/gda-tds-recordset.c:450
msgid "could not allocate data holder"
msgstr "não pôde alocar detentor de dados"

#: providers/gda-sybase-server/gda-sybase-recordset.c:467
#: providers/gda-tds-server/gda-tds-recordset.c:463
msgid "could not ct_bind data holder to column"
msgstr "não pôde ct_bind detentor de dados para coluna"

#: providers/gda-sybase-server/gda-sybase-recordset.c:474
#: providers/gda-tds-server/gda-tds-recordset.c:470
msgid "illegal type detected, aborting"
msgstr "tipo ilegal detectado, abortando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:481
#: providers/gda-tds-server/gda-tds-recordset.c:477
msgid "could not allocate field"
msgstr "não pôde alocar campo"

#: providers/gda-sybase-server/main-sybase.c:78
msgid "Could not register gda-sybase provider implementation"
msgstr "Não pôde registrar implementação de provedor gda-sybase"

#: providers/gda-primebase-server/GNOME_Database_Provider_Primebase.server.in.h:1
msgid "GDA Datasource Access for primebase"
msgstr ""

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-primebase-server/gda-primebase-connection.c:157
#, fuzzy, c-format
msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
msgstr "não pôde abrir conexão para %s"

#: providers/gda-primebase-server/gda-primebase-connection.c:163
#, c-format
msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:171
msgid "Could not get PBIConnectionInfo(), disconnecting"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:176
#, c-format
msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:390
#, fuzzy
msgid "Not enough memory for primebase_Error structure."
msgstr "Sem memória suficiente para conexão com o sybase"

#: providers/gda-primebase-server/gda-primebase-connection.c:397
#, c-format
msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
msgstr ""

#: providers/gda-primebase-server/main-primebase.c:73
#, fuzzy
msgid "Could not register PRIMEBASE provider implementation"
msgstr "Não pôde registrar implementação de provedor INTERBASE"

#: providers/gda-tds-server/GNOME_Database_Provider_TDS.server.in.h:1
msgid "GDA Datasource Access for Sybase/MS SQL"
msgstr ""

#: providers/gda-tds-server/GNOME_Database_Provider_TDS.server.in.h:2
msgid "TDS provider connection factory"
msgstr ""

#: providers/gda-tds-server/gda-tds-connection.c:160
#, fuzzy
msgid "Not enough memory for tds_Connection"
msgstr "Sem memória suficiente para conexão com o sybase"

#. Fixme: replace "dummy" with buf when CS_GET CS_SERVERNAME works
#: providers/gda-tds-server/gda-tds-connection.c:888
#, fuzzy, c-format
msgid "Reconnected to %s"
msgstr "Reconectado para '%s'"

#: providers/gda-tds-server/gda-tds-error.c:70
#, fuzzy, c-format
msgid "gda_tds_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-tds-server/main-tds.c:78
#, fuzzy
msgid "Could not register gda-tds provider implementation"
msgstr "Não pôde registrar implementação de provedor gda-sybase"

#: tools/gda-run.c:102
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: entrada DSN mal configurada"

#: tools/gda-run.c:116
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: fonte de dados '%s' não encontrada\n"

#: tools/templates/main-srv.c.tmpl:60
#, fuzzy
msgid "Could not register <PROVIDER-NAME> provider implementation"
msgstr "Não pôde registrar implementação de provedor INTERBASE"

#~ msgid "%s: The server didn't respond."
#~ msgstr "%s: O servidor não respondeu."

#~ msgid "thread is already running"
#~ msgstr "encadeamento já está sendo executado"

#~ msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
#~ msgstr "impl_GDA_Connection_open: abrindo conexão com DSN: %s"

#~ msgid "%s: an error ocurred while fetching data"
#~ msgstr "%s: um erro ocorreu enquanto obtendo dados"

#~ msgid ""
#~ "OAF doesn't know about our IID; indicates broken installation. Exiting..."
#~ msgstr ""
#~ "OAF não sabe a respeito sobre IID; indica instalação interrompida. "
#~ "Saindo..."

#~ msgid ""
#~ "Another instance of this provider is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Uma outra instância deste provedor já está registrada com OAF. Saindo..."

#~ msgid "Unknown error registering this provider with OAF. Exiting"
#~ msgstr "Erro desconhecido registrando este provedor com OAF. Saindo."

#~ msgid "Registered with ID = %s"
#~ msgstr "Registrado com ID = %s"

#~ msgid "CORBA exception unloading report engine: %s"
#~ msgstr "Exceção CORBA descarregando mecanismo de relatório: %s"

#~ msgid "Received signal %d, shutting down."
#~ msgstr "Sinal recebido %d, desligando."

#~ msgid "Received signal %d, shutting down cleanly"
#~ msgstr "Sinal recebido %d, desligando"

#~ msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
#~ msgstr ""
#~ "Falhou ao inicializar OAF: favor enviar relatório de falha para "
#~ "osmantenedores OAF"

#~ msgid "Could not activate report engine"
#~ msgstr "Não pôde ativar mecanismo de relatório"

#~ msgid ""
#~ "Another instance of the report engine is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Outra instância do mecanismo de relatório já registrado com OAF. Saindo..."

#~ msgid "Unknown error registering report engine with OAF. Exiting"
#~ msgstr ""
#~ "Erro desconhecido registrando mecanismo de relatório com OAF. Saindo"

#~ msgid "Registered report engine with ID = %s"
#~ msgstr "Mecanismo de relatório registrado com identificação = %s"

#~ msgid "%s: length = %d"
#~ msgstr "%s: extensão = %d"

#~ msgid "%s: sending field description for '%s'"
#~ msgstr "%s: enviando descrição de campo para '%s'"

#~ msgid "Error allocating error structure"
#~ msgstr "Erro ao alocar estrutura de erro"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: nome de tabela = '%s'"

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: tabela_esquema = '%s'"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
#~ msgstr "chema_procedures: proc_catalog = '%s' NÃO USADO!\n"

#~ msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
#~ msgstr "schema_procedures: proc_catalog = '%s' NÃO USADO!"

# fe/fe_main.c:186
#~ msgid "database name"
#~ msgstr "nome da base de dados"

#~ msgid "DATABASE"
#~ msgstr "BASE DE DADOS"

# fe/fe_impexp.c:164 fe/fe_impexp.c:274 fe/fe_impexp.c:391
# gda-clnt-ui/gnome-db-browser.c:217
#~ msgid "user name"
#~ msgstr "nome de usuário"

#~ msgid "USER-NAME"
#~ msgstr "NOME-USUÁRIO"

#~ msgid "password"
#~ msgstr "senha"

#~ msgid "PASSWORD"
#~ msgstr "SENHA"

# fe/fe_impexp.c:164 fe/fe_impexp.c:274 fe/fe_impexp.c:391
# gda-clnt-ui/gnome-db-browser.c:217
#~ msgid "file name"
#~ msgstr "nome de arquivo"

#~ msgid "FILE-NAME"
#~ msgstr "NOME-ARQUIVO"

#~ msgid "Fetchong %d cslib message"
#~ msgstr "Recuperando %d mensagem cslib"

#~ msgid "%s: not implemented"
#~ msgstr "%s: não implementado"
